locales/de/subtitles/erwin-schulz/erwin-schulz-02.yml

```diff
@@ -157,51 +154,49 @@ de:
           s: 30
           from: 00:02:43,29
           to: 00:02:47,08
-          text: I wanted to become a type setter.
+          text: Also, ich wollte da Schriftsetzer werden.
         sequence-31:
           s: 31
           from: 00:02:47,08
           to: 00:02:50,26
-          text: My uncle sought for a place – rejected.
+          text: Mein Onkel hat sich auch drum bemüht – abgelehnt.
         sequence-32:
           s: 32
           from: 00:02:50,26
           to: 00:02:57,02
-          text: 'Then another uncle said: “Well, then you become a bricklayer” – as
-            well, no success.'
+          text: Dann hat ein anderer Onkel gesagt: „Na, dann wirst du Maurer - auch nichts.
         sequence-33:
           s: 33
           from: 00:02:57,02
           to: 00:03:06,17
-          text: My father was a metal worker – again, nothing.
+          text: Mein Vater war Schlosser – auch nichts. Also, alle Bemühungen sind sozusagen fehlgeschlagen.
         sequence-34:
           s: 34
           from: 00:03:06,17
-          to: 00:03:23,03
-          text: And then I started at the company ‘Jonas’ as an adolescent employee.
+          to: 00:03:18,03
+          text: Und dann fing ich an bei der Firma Jonas als jugendlicher Angestellter. Ich war dann in der Zeit
         sequence-35:
           s: 35
-          from: 00:03:23,03
+          from: 00:03:18,03
           to: 00:03:25,16
-          text: That was in 1927.
+          text: – na, ein besserer Laufbursche. Für ’ne gewisse Zeit. Das war 1927.
```

Comment:
Beginning the subtitle with a dash looks kind of weird. I would just skip the dash or maybe even the dash and the 'na' and start with `ein besserer...`. What do you think?